drivers/SmartThings/philips-hue/src/disco/light.lua

```diff
@@ -95,6 +89,7 @@ function M.handle_discovered_device(
         parent_assigned_child_key = parent_assigned_child_key
       }
 
+      log.debug(true, st_utils.stringify_table(st_metadata, "light create", true))
```

Comment:
debug statement

Reply:
I think I actually want to leave these in until after QA/SVT is wrapped.